spark/common/tools/maven/scalastyle_config.xml

```diff
@@ -0,0 +1,194 @@
+<scalastyle commentFilter="enabled">
```

Comment:
Why put this in `spark/common` alone? What about other scala projects under `spark` such as `spark-3.4` `spark-3.5` ...